index.html

```diff
@@ -783,6 +786,32 @@ <h5>Support for unstructured grids</h5>
     </div>
   </div>
 
+  <hr>
+  <h2 id="gallery">Gallery</h2>
+
+  <h3 id="bathymetry">Bathymetric</h3>
```

Comment:
Not sure this title is needed right now. When we get to a very large set we can think again about dividing in categories, but for now let's not do that?
```suggestion
```

Reply:
agreed. We can categorize it later. In today's OWM, we just created this one, first example, and I made the binder connection (thanks to @willirath - took your parcels binder as a template) so we can see the first result. Next steps are to populate the gallery repo and then we can come up with a structure when we have a better sample and impression of the kind of gallery objects.